src/main/java/com/tripply/Auth/entity/PasswordResetToken.java

```diff
@@ -0,0 +1,50 @@
+package com.tripply.Auth.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+@Entity
+@Table(name = "password_reset_token", schema = "identity")
+public class PasswordResetToken {
+
+    private static final int EXPIRATION = 60 * 1;
```

Comment:
Take this value from configuration file.

Reply:
Resolved